src/PKSim.Matlab/MatlabIndividualFactory.cs

```diff
@@ -65,7 +66,13 @@ public ParameterValue[] CreateIndividual(OriginData matlabOriginData, IEnumerabl
          return allIndividualParameters.ToArray();
       }
 
-      public DistributedParameterValue[] DistributionsFor(OriginData matlabOriginData, IEnumerable<MoleculeOntogeny> moleculeOntogenies)
+      public ParameterValue[] CreateIndividual(OriginData matlabOriginData)
+      {
+         IEnumerable<MoleculeOntogeny> moleculeOnogenies = new MoleculeOntogeny[0];
+         return CreateIndividual(matlabOriginData, moleculeOnogenies);
```

Comment:
If my suggestion from above does not work, simply pass null as an argument
```
return CreateIndividual(matlabOriginData, null);
```

Reply:
and change the implementation of `https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/102d271e84823151de2ae9449d5779b51f1cb809/src/PKSim.Matlab/OntogenyFactorsRetriever.cs#L47`
to deal with null as expected.

e.g. if the list of ontogeny is null return

```
      if (!allOntogeniesForSpecies.Any() || moleculeOntogenies==null)
            return allOntogenyFactors;
```

Reply:
@PavelBal 
Forget about that. We need to merge something to see if it works with R and Matlab while we still support both

Suggestion
REmplace 
```
IEnumerable<MoleculeOntogeny> moleculeOnogenies = new MoleculeOntogeny[0];
return CreateIndividual(matlabOriginData, moleculeOnogenies);
```

with 
```
return CreateIndividual(matlabOriginData, Enumerable.Empty<MoleculeOntogeny>());
```

and I'll merge

Reply:
Updated.